# -*- coding:utf-8 -*-

from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException
from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException

# module saklient.cloud.errors.servercouldnotstopexception

class ServerCouldNotStopException(HttpServiceUnavailableException):
    ## サービスが利用できません。サーバを停止できません。再度お試しください。
    
    # (class field) default_message = "サービスが利用できません。サーバを停止できません。再度お試しください。"
    
    pass
